modules/rtbsolutionsBidAdapter.js

```diff
@@ -0,0 +1,99 @@
+import { registerBidder } from '../src/adapters/bidderFactory';
+import * as utils from '../src/utils';
+
+const BIDDER_CODE = 'rtbsolutions';
+const ENDPOINT_URL = 'https://dsp-eu-lb.rtbsolutions.pro/bid/hb';
+
+export const spec = {
+  version: '1.0',
+  code: BIDDER_CODE,
+  aliases: ['rtbss'], // short code
+  nurls: {},
+  isBidRequestValid: function(bid) {
+    return !!bid.params.blockId;
+  },
+  buildRequests: function(validBidRequests, bidderRequest) {
+    let req = [];
+
+    bidderRequest.bids.forEach(item => {
+      const width = item.sizes[0][0];
+      const height = item.sizes[0][1];
+
+      let imp = {
+        referer: bidderRequest.refererInfo.referer,
+        ua: navigator.userAgent,
+        lang: this.getLanguage(),
+        domain: this.getDomain(),
+        width: width,
+        height: height,
+        type: 'banner',
+      };
+
+      if (item.params.s1 !== undefined) imp.s1 = item.params.s1;
+      if (item.params.s2 !== undefined) imp.s2 = item.params.s2;
+      if (item.params.s3 !== undefined) imp.s3 = item.params.s3;
+      if (item.params.s4 !== undefined) imp.s4 = item.params.s4;
+
+      req.push({
+        bid_id: item.bidId,
+        block_id: item.params.blockId,
+        ver: this.version,
+        imp
+      });
+    });
+
+    return {
+      method: 'POST',
+      url: ENDPOINT_URL,
+      data: req,
+      options: {
+        contentType: 'application/json'
+      }
+    }
+  },
+  interpretResponse: function(serverResponse, request) {
+    const bidResponses = [];
+
+    serverResponse.body.forEach(item => {
+      this.nurls[item.bid_id] = item.nurl;
+
+      const bidResponse = {
+        requestId: item.bid_id,
+        cpm: item.cpm,
+        width: item.width,
+        height: item.height,
+        creativeId: item.creative_id,
+        currency: item.currency,
+        netRevenue: true,
+        ttl: 360,
+        ad: item.ad,
+      };
+
+      bidResponses.push(bidResponse);
+    });
+
+    return bidResponses;
+  },
+  onBidWon: function(bid) {
+    fetch(this.nurls[bid.requestId])
```

Comment:
Can you use the `ajax` interface from `src/ajax.js` instead of the `fetch` here to make your follow-up call to your endpoint?  Prebid.js doesn't support Promises at this time.

Reply:
Hi @jsnellbaker , thanks for your review, we have submitted the update